app/controllers/api/authentications_controller.rb

```diff
@@ -1,4 +1,15 @@
 module Api
   class AuthenticationsController < BaseController
+    def options
+      render_options(:authentications, build_additional_fields)
+    end
+
+    private
+
+    def build_additional_fields
+      {
+        :cloud_types => ManageIQ::Providers::AnsibleTower::AutomationManager::CloudCredential.credential_types
```

Comment:
they looked sorted, but that may not always be guaranteed.  can you add a sort here or in the credential_types method itself (otherwise, you'd need the sort in the rspec too).